docs/rules/no-thenable.md

```diff
@@ -0,0 +1,63 @@
+# Disallow `then` property
+
+✅ *This rule is part of the [recommended](https://github.com/sindresorhus/eslint-plugin-unicorn#recommended-config) config.*
+
+If an object is defined as "thenable", once it's accidentally used in an await expression, it may cause problems.
```

Comment:
Any docs to link to about this for context?

Reply:
I can't find any link for this.